guides/doc-Provisioning_Guide/topics/Cloud-Azure.adoc

```diff
@@ -0,0 +1,258 @@
+--compute-resource _azure_cr_name_ \                               <2>
+--uuid '_RedHat:RHEL:7-RAW:latest_' \                              <3>
+--username '_azure_username_' \                                    <4>
+--user-data _no_                                                   <5>
+----
+<1> With the `--name` option, enter a name for the image.
+<2> With the `--compute-resource` option, enter a name of the Azure Compute resource that you want to create an image for.
+<3> With the `--uuid` option, enter a Marketplace URN. For example, *RedHat:RHEL:7-RAW:latest*. For more information, see https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage[Find Linux VM images in the Azure Marketplace with the Azure CLI].
+<4> With the `--username` option, enter the SSH user name for image access. Note that the username that you enter for the image must be the same that you use when creating a host. The `--password` option is optional when creating an image. You cannot use the `root` user.
+<5> With the `--user-data` option, define whether or not the image supports user data.
+
+
+[[Provisioning_Virtual_Machines_in_Azure-Adding_Azure_Details_to_a_Compute_Profile]]
+=== Adding Azure Details to a Compute Profile
+
+You can add Azure details to a compute profile to automatically populate virtual machine-based settings on host creation. For more information about Azure Resource Manager concepts, see https://docs.microsoft.com/en-us/azure/azure-resource-manager/[Azure Resource Manager documentation].
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Infrastructure* > *Compute Profiles* and click the name of a compute profile to use.
+. Select an Azure connection.
+. From the *Resource group* list, select the resource group to provision to. Click *Reload Images, Sizes, vNets* to load information from Azure RM.
+. From the *VM Size* list, select a VM size setting.
+. From the *Platform* list, select *Linux*. Satellite does not support provisioning Windows VMs. 
+. In the *Username* field, enter a user name to authenticate with. Note that the username that you enter for compute profile must be the same that you use when creating an image.
+. To authenticate the user, use one of the following options:
++
+* To authenticate using a password, enter a password in the *Password* field.
++
+* To authenticate using an SSH key, enter an SSH key inn the *SSH Key* field.
+. Optional: If you the VM to use a premium SSD, select the *Premium OS Disk* check box.
+. From the *OS Disk Caching* list, select the disc caching setting.
+. Optional: In the *Custom Script Command* field, enter commands to perform on the VM when the VM is provisioned.
+. Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use. {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host.
```

Comment:
Could you also mention to prepend the  **Custom Script Command** with `sudo` when user wants to give a value to **Comma separated file URIs** since the file will be downloaded to `/var/lib/waagent/custom-script/download/0/` and for user to actually execute that file, they will need `sudo`.